src/style.css

```diff
@@ -2,0 +1,39 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  padding-right: 5px;
+  width: 96px;
+  justify-content: space-between;
+}
+
+.stars__star {
+  display: flex;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+}
+
+.stars--1 :nth-child(1) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 :nth-child(-1n + 2) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--3 :nth-child(-1n + 3) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 :nth-child(-1n + 4) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--5 :nth-child(-1n + 5) {
+  background-image: url(images/star-active.svg);
```

Comment:
You are using the descendant combinator to apply styles to child elements, which could lead to performance issues if overused. In this case, it's better to apply the active star style directly to the elements that need it by assigning a specific class to active stars, rather than using the `:nth-child` pseudo-class selector.